.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+labels: Bug
+
+---
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is, and what commands you ran.]
+
+
+**What did you expect to happen:**
+
+
+**The output of the following commands and cstor pod logs will help us better understand what's going on**:
+(Pasting long output into a [GitHub gist](https://gist.github.com) or other [Pastebin](https://pastebin.com/) is fine.)
+
+* `kubectl logs deployment/maya-apiserver -n openebs`
+* `kubectl logs deployment/<pv-name>-target -n <openebs_namespace> -c cstor-istgt`
+* `kubectl get pods -n openebs`
+* `kubectl get cvr,spc,csp,cstorvolume -A -o yaml`
```

Comment:
For reporting issue, I think we need logs of cstor-pool container also

Reply:
yes, i've mentioned in above comment.

Reply:
that is target pod istgt container right?

Reply:
https://github.com/openebs/cstor/pull/301/files/a7d4c5c80c518e44fbb2cfcf890a5bfafa8dfb40#diff-74c2649c999338ed7a2833d79d4b25c3R15

Reply:
Ok!! I am searching for command...